test/test_node.go

```diff
@@ -266,46 +268,43 @@ func isReachable(wg *sync.WaitGroup, t *testing.T, host ContainerPort, ch chan<-
 	ch <- connectionAttempt(t, host)
 }
 
-func (hosts Hosts) WaitForAllToStart(t *testing.T, timeout int) {
+func (hosts Hosts) WaitForAllToStart(t *testing.T, timeout int) error {
```

Comment:
When would you want this to return an error without failing the test?

Reply:
same as TestSigner